### DISCUSSION TITLE
apple/firmware/im191.fd

### DISCUSSION THREAD

**Discussion by markwbennett** [baughmann/designaire-z390-intel-i9-9900k-opencore#82] [General]
Thank you for sharing. My EFI includes (as well as OC and BOOT) a folder called APPLE with  a subfolder FIRMWARE containing a 16.8MB file, IM191.fd.

Is this something that should be there? My EFI has gone through many iterations, and I don't remember putting this in it, but it's dated 5/31/24.

Thank you.

**Comment by seven-of-eleven** [comment]
I typically delete the “APPLE” folders that appear in my EFI. I haven’t had an issue from deleting them.
